### DISCUSSION TITLE
Why tags won't is there (yet)?

### DISCUSSION THREAD

**Discussion by hari-bo** [enchant97/note-mark#190] [Ideas]
Within the roadmap i saw that tags functionality wont be added. But when adding tags this note taking solution is complete with every note (in my opinion) must have features notebooks/links and tags.

**Comment by enchant97** [comment]
I don't see how it will aid me in my own note taking. Where do you see tags being utilised in your workflow?

**Reply by hari-bo** [reply]
In common note taking solutions like Evernote, Obsidian etc you can place an tag as well. I'm using tags myself for specify posts with an specific topic within an notebook group instead of micromanaging an create a net notebook group.

**Comment by riker09** [comment]
I was about to open my own feature request / discussion on tags when i discovered this discussion. 😅 

Tags are a different way of organizing things. I like to use them as they allow me to put a "thing" into more than one category. Take notes, for example. Folders are nice, but a note can only be in one folder at a time (not taking symlinks into account).

A simple use case would involve the three tags #todo, #work and #private. If I wanted to get an overview over all my #todos tags would allow me to do so.

**Reply by enchant97** [reply]
@riker09 Feel free to open a feature request. They should be possible to implement in a future V1 update!
